src/auditlog/registry.py

```diff
@@ -92,7 +98,28 @@ def _connect_signals(self, model):
         """
         for signal in self._signals:
             receiver = self._signals[signal]
-            signal.connect(receiver, sender=model, dispatch_uid=self._dispatch_uid(signal, model))
+
+            if signal is m2m_changed:
```

Comment:
This special logic doesn't have its counterpart in `_disconnect_signals`. Disconnect just won't work for m2m signals.